test/support/swagger_example.json

```diff
@@ -65,7 +65,8 @@
         "security": [{
           "basic": []
         }],
-        "summary": ""
+        "summary": "",
```

Comment:
@brosquinha In the documentation I got the impression that we also need to describe the available tags, before using. Did you got a chance that just adding the tags to each example would be enough?

Reply:
Yep, I was also worried that this would be necessary to make the grouping work. However, I tested the swagger.json with [Swagger Editor](https://editor.swagger.io/) and the result was valid and the grouping worked as intended.

Reply:
:heart: Nice!